GPU/Common/FramebufferManagerCommon.cpp

```diff
@@ -523,16 +524,17 @@ void FramebufferManagerCommon::CopyToDepthFromOverlappingFramebuffers(VirtualFra
 	};
 
 	std::vector<CopySource> sources;
-	for (auto src: vfbs_) {
+	for (auto src : vfbs_) {
 		if (src == dest)
 			continue;
 
 		if (src->fb_address == dest->z_address && src->fb_stride == dest->z_stride && src->format == GE_FORMAT_565) {
-			if (src->colorBindSeq < dest->depthBindSeq) {
+			if (src->colorBindSeq > dest->depthBindSeq) {
 				// Source has newer data than the current buffer, use it.
 				sources.push_back(CopySource{ src, RASTER_COLOR });
 			}
 		} else if (src->z_address == dest->z_address && src->z_stride == dest->z_stride && src->depthBindSeq > dest->depthBindSeq) {
+			// Don't bother if the buffer was from another frame. This heuristic is old.
```

Comment:
Should we not bother?  Although most games re-render depth every frame, it wouldn't shock me for a game to keep it around for render-to-texture in a dialog scene (i.e. a frozen image behind, blurred using depth.)  Not sure if any do that, though.

But I think this comment claims something that isn't currently done anyway.

-[Unknown]

Reply:
Hm that's right, this comment was wrong and based on a misunderstanding I had for a while. I'll clean it up.